app/models/miq_task.rb

```diff
@@ -33,6 +33,17 @@ class MiqTask < ApplicationRecord
     t.grouping(Arel::Nodes::Case.new(t[:state]).when(STATE_FINISHED).then(t[:status]).else(t[:state]))
   end)
 
+  scope :active, -> { where(:state => STATE_ACTIVE) }
+  scope :no_associated_job, -> { where.not("id IN (SELECT miq_task_id from jobs)") }
+  scope :timed_out, -> { where("updated_on < ?", Time.now.utc - ::Settings.task.active_task_timeout.to_i_with_method) }
+
+  def self.update_status_for_timed_out_active_tasks
+    MiqTask.active.timed_out.no_associated_job.find_each do |task|
+      task.update_status(STATE_FINISHED, STATUS_ERROR,
```

Comment:
`Timeout` not present as possible state and `task` with status `Timeout` will not be shown on Task management screens:

<img width="538" alt="screen shot 2017-05-26 at 1 40 56 pm" src="https://cloud.githubusercontent.com/assets/6556758/26506012/fb53de34-4218-11e7-8d1a-69640f53610e.png">
I think we should add `Timeout` to a list of possible state filter.

@gtanzillo what do you think,  should we first allow timed out tasks to be visible on task management screens and change this PR to set status to  `STATUS_TIMEOUT` ?
or proceed with with this PR setting timed out task `Error` status ?
or set status to `Warn` ?